[Bug]: privacy.sexy@0.12.1: download via aria2 failed

### Prerequisites

- [X] I have written a descriptive issue title.
- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.
- [X] I have verified that I am using the latest version of Scoop and corresponding bucket.

### Package Name

privacy.sexy

### Expected/Current Behaviour

PS C:\Users\XXX> scoop update privacy.sexy
privacy.sexy: 0.12.0 -> 0.12.1
Updating one outdated app:
Updating 'privacy.sexy' (0.12.0 -> 0.12.1)
Downloading new version
Starting download with aria2 ...
...............................................................
Download: (OK):download completed.

### Steps to Reproduce

```console
PS C:\Users\XXX> scoop update privacy.sexy
privacy.sexy: 0.12.0 -> 0.12.1
Updating one outdated app:
Updating 'privacy.sexy' (0.12.0 -> 0.12.1)
Downloading new version
Starting download with aria2 ...
Download: 08/20 13:55:09 [ERROR] CUID#8 - Download aborted. URI=https://github.com/undergroundwires/privacy.sexy/releaseDownload: Exception: [AbstractCommand.cc:351] errorCode=3 URI=https://github.com/undergroundwires/privacy.sexy/releases/Download: Download Results:
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: 59d0a7|ERR |       0B/s|C:/Users/XXX/scoop/cache/privacy.sexy#0.12.1#https_github.com_undergroundwires_privacy.sexy_releases_download_0.12.1_privacy.sexy-Setup-0.12.1.exe_dl.7z
Download: Status Legend:
Download: (ERR):error occurred.
Download: aria2 will resume download if the transfer is restarted.
Download: If there are any errors, then see the log file. See '-l' option in help/man page for details.

ERROR Download failed! (Error 3) Resource was not found
ERROR https://github.com/undergroundwires/privacy.sexy/releases/download/0.12.1/privacy.sexy-Setup-0.12.1.exe#/dl.7z
    referer=https://github.com/undergroundwires/privacy.sexy/releases/download/0.12.1/privacy.sexy-Setup-0.12.1.exe#/
    dir=C:\Users\kiwap\scoop\cache
    out=privacy.sexy#0.12.1#https_github.com_undergroundwires_privacy.sexy_releases_download_0.12.1_privacy.sexy-Setup-0.12.1.exe_dl.7z

ERROR & 'C:\Users\XXX\scoop\apps\aria2\current\aria2c.exe' --input-file='C:\Users\XXX\scoop\cache\privacy.sexy.txt' --user-agent='Scoop/1.0 (+http://scoop.sh/) PowerShell/7.3 (Windows NT 10.0; Win64; x64; Core)' --allow-overwrite=true --auto-file-renaming=false --retry-wait=2 --split=5 --max-connection-per-server=5 --min-split-size=5M --console-log-level=warn --enable-color=false --no-conf=true --follow-metalink=true --metalink-preferred-protocol=https --min-tls-version=TLSv1.2 --stop-with-process=12888 --continue --summary-interval=0 --auto-save-interval=1

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/ScoopInstaller/Extras/issues/new?title=privacy.sexy%400.12.1%3a+download+via+aria2+failed
PS C:\Users\XXX>
```


### Possible Solution

The package binary naming scheme was changed.

For previous release, the name was [privacy.sexy-Setup-0.12.0.exe](https://github.com/undergroundwires/privacy.sexy/releases/download/0.12.0/privacy.sexy-Setup-0.12.0.exe) with dashes, however, the name is 
[privacy.sexy.Setup.0.12.1.exe](https://github.com/undergroundwires/privacy.sexy/releases/download/0.12.1/privacy.sexy.Setup.0.12.1.exe) with no dash for current release.

### Scoop and Buckets Version

```console
PS C:\Users\XXX> scoop --version
Current Scoop version:
v0.3.1 - Released at 2022-11-15

'extras' bucket:
1175f646c (HEAD -> master, origin/master, origin/HEAD) wsl2-distro-manager: Update to version 1.8.8

'java' bucket:
27fa4fd0 (HEAD -> master, origin/master, origin/HEAD) temurin8-nightly-jre: Update to version 8.0.392-1.0.202308191457

'jbangdev' bucket:
749e3f6 (HEAD -> main, tag: v0.110.1, origin/main, origin/HEAD) jbang v0.110.1

'main' bucket:
0eb2f7685 (HEAD -> master, origin/master, origin/HEAD) intermodal: Update to version 0.1.13
```


### Scoop Config

```console
PS C:\Users\XXX> scoop config

aria2-warning-enabled last_update          scoop_repo                              scoop_branch
--------------------- -----------          ----------                              ------------
                False 8/20/2023 2:07:52 PM https://github.com/ScoopInstaller/Scoop master
```


### PowerShell Version

```console
PS C:\Users\XXX> $PSVersionTable

Name                           Value
----                           -----
PSVersion                      7.3.6
PSEdition                      Core
GitCommitId                    7.3.6
OS                             Microsoft Windows 10.0.22621
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```


### Additional Softwares

_No response_

--- a/bucket/privacy.sexy.json
+++ b/bucket/privacy.sexy.json
@@ -5,8 +5,8 @@
     "license": "GPL-3.0",
     "architecture": {
         "64bit": {
-            "url": "https://github.com/undergroundwires/privacy.sexy/releases/download/0.12.1/privacy.sexy-Setup-0.12.1.exe#/dl.7z",
-            "hash": "sha512:82874873939d0ad8458f15c85cc766e6525f93c77a3d05f1ccd8cc9837bb5e4f96f791167fadf886c8166862ea3d2347ff529994bc5414eb3d34d0526b379574"
+            "url": "https://github.com/undergroundwires/privacy.sexy/releases/download/0.12.1/privacy.sexy.Setup.0.12.1.exe#/dl.7z",
+            "hash": "sha512:bd446c36398900620c487f88b84a682e0cdc4acc85088091de8e57862c7b49f3cd23903fcbdadcb5f77f69a6e5ff919efa21b42c2c03722157206fb2477732d1"
         }
     },
     "extract_dir": "$PLUGINSDIR",
@@ -24,7 +24,7 @@
     "autoupdate": {
         "architecture": {
             "64bit": {
-                "url": "https://github.com/undergroundwires/privacy.sexy/releases/download/$version/privacy.sexy-Setup-$version.exe#/dl.7z",
+                "url": "https://github.com/undergroundwires/privacy.sexy/releases/download/$version/privacy.sexy.Setup.$version.exe#/dl.7z",
                 "hash": {
                     "url": "$baseurl/latest.yml",
                     "regex": "sha512: $base64"

